sunpy/timeseries/timeseries_factory.py

```diff
@@ -475,7 +475,6 @@ def __call__(self, *args, **kwargs):
         new_timeseries += already_timeseries
 
         # Concatenate the timeseries into one if specified.
```

Comment:
you appear to have moved the line of code but not the comment.

Reply:
This still comments the actual code where concatenate gets done, I've just moved the kwarg popping to avoid un-wanted kwargs being handed on at some point.